docs/spdx_output_golang_1.12_alpine.txt

```diff
@@ -0,0 +1,337 @@
+
+PackageName: 23419e18686687dd37c97486ac9752f59fbc2d3e22f3c5272aeea632d439bc15
+PackageChecksum: SHA256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+SPDXID: SPDXRef-23419e1868
+PackageDownloadLocation: golang:1.12-alpine
+FilesAnalyzed: false
+PackageComment: <text>Layer: e3b0c44298:
+	info: Instruction Line: RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+	warning: 
+Unrecognized Commands:mkdir -p "$GOPATH/src" "$GOPATH/bin"
+chmod -R 777 "$GOPATH"
+
+</text>
+
+Relationship: SPDXRef-23419e1868 HAS_PREREQUISITE SPDXRef-1f6ba46a9c
+
+PackageName: musl
+PackageVersion: musl-1.1.20-r4
+PackageLicenseDeclared: MIT
+PackageCopyrightText: NOASSERTION
+PackageDownloadLocation: NOASSERTION
+SPDXID: SPDXRef-musl.musl-1.1.20-r4
+FilesAnalyzed: false
+PackageComment: <text>musl:
+	warning: No metadata for key: copyright
+	warning: No metadata for key: download_url
+</text>
+
+PackageName: busybox
+PackageVersion: busybox-1.29.3-r10
+PackageLicenseDeclared: GPL-2.0
+PackageCopyrightText: NOASSERTION
+PackageDownloadLocation: NOASSERTION
+SPDXID: SPDXRef-busybox.busybox-1.29.3-r10
+FilesAnalyzed: false
+PackageComment: <text>busybox:
+	warning: No metadata for key: copyright
+	warning: No metadata for key: download_url
+</text>
+
+PackageName: alpine-baselayout
+PackageVersion: alpine-baselayout-3.1.0-r3
+PackageLicenseDeclared: GPL-2.0
+PackageCopyrightText: NOASSERTION
+PackageDownloadLocation: NOASSERTION
+SPDXID: SPDXRef-alpine-baselayout.alpine-baselayout-3.1.0-r3
+FilesAnalyzed: false
+PackageComment: <text>alpine-baselayout:
+	warning: No metadata for key: copyright
+	warning: No metadata for key: download_url
+</text>
+
+PackageName: alpine-keys
+PackageVersion: alpine-keys-2.1-r1
+PackageLicenseDeclared: MIT
+PackageCopyrightText: NOASSERTION
+PackageDownloadLocation: NOASSERTION
+SPDXID: SPDXRef-alpine-keys.alpine-keys-2.1-r1
+FilesAnalyzed: false
+PackageComment: <text>alpine-keys:
+	warning: No metadata for key: copyright
+	warning: No metadata for key: download_url
+</text>
+
+PackageName: libcrypto1.1
+PackageVersion: libcrypto1.1-1.1.1b-r1
+PackageLicenseDeclared: OpenSSL
+PackageCopyrightText: NOASSERTION
+PackageDownloadLocation: NOASSERTION
+SPDXID: SPDXRef-libcrypto1.1.libcrypto1.1-1.1.1b-r1
+FilesAnalyzed: false
+PackageComment: <text>libcrypto1.1:
+	warning: No metadata for key: copyright
+	warning: No metadata for key: download_url
+</text>
+
+PackageName: libssl1.1
+PackageVersion: libssl1.1-1.1.1b-r1
+PackageLicenseDeclared: OpenSSL
+PackageCopyrightText: NOASSERTION
+PackageDownloadLocation: NOASSERTION
+SPDXID: SPDXRef-libssl1.1.libssl1.1-1.1.1b-r1
+FilesAnalyzed: false
+PackageComment: <text>libssl1.1:
+	warning: No metadata for key: copyright
+	warning: No metadata for key: download_url
+</text>
+
+PackageName: ca-certificates-cacert
+PackageVersion: ca-certificates-cacert-20190108-r0
+PackageLicenseDeclared: MPL-2.0 GPL-2.0-or-later
```

Comment:
There is a missing operand between these two licenses - an AND is required if this is a conjunctive license where the terms of both licenses apply or an OR is required if this is a disjunctive license where there is a choice of license terms